File: populate-test-data.js
#!/usr/bin/env node

// Simple script to populate test data
// Run this with: node populate-test-data.js

const baseUrl = 'http://localhost:3000';

async function populateTestData() {
  try {
    console.log('🎾 Populating Padel courts with random test data...');

    const response = await fetch(`${baseUrl}/api/populate-test-data`, {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
      },
    });

    const result = await response.json();

    if (result.success) {
      console.log('✅ Success!', result.message);
      console.log('🕒 Timestamp:', result.timestamp);
      console.log('\n🏆 All courts now have random team names and scores!');
      console.log('📱 Check your app at http://localhost:3000');
    } else {
      console.error('❌ Error:', result.error);
    }
  } catch (error) {
    console.error('❌ Network error:', error.message);
    console.log('\n💡 Make sure your Next.js app is running on http://localhost:3000');
  }
}

// Run the script
populateTestData();

File: populate-test-data.py
#!/usr/bin/env python3

import requests
import json
import sys

def populate_test_data(base_url="http://localhost:3000"):
    """Populate Padel courts with random test data"""

    print("🎾 Populating Padel courts with random test data...")

    try:
        response = requests.post(
            f"{base_url}/api/populate-test-data",
            headers={"Content-Type": "application/json"}
        )

        result = response.json()

        if result.get("success"):
            print(f"✅ Success! {result.get('message')}")
            print(f"🕒 Timestamp: {result.get('timestamp')}")
            print("\n🏆 All courts now have random team names and scores!")
            print(f"📱 Check your app at {base_url}")
        else:
            print(f"❌ Error: {result.get('error')}")
            return False

    except requests.exceptions.ConnectionError:
        print(f"❌ Connection error: Could not connect to {base_url}")
        print("💡 Make sure your Next.js app is running")
        return False
    except Exception as e:
        print(f"❌ Error: {str(e)}")
        return False

    return True

if __name__ == "__main__":
    # Allow custom URL as command line argument
    url = sys.argv[1] if len(sys.argv) > 1 else "http://localhost:3000"
    populate_test_data(url)